design/ocwg-design-decisions.md

```diff
@@ -1,92 +1,42 @@
-resource are (a) dynamic (e.g. a weather forecast), or (b) large (e.g.
-videos) so that downloading is not always the best strategy.
-
-## Graceful Degration
-
-We want fall-backs for difficult content types. In email, HTML mails
-usually have a plain-text fall-back.
-
-## Extensible
-
-We want a number of apps, including those not known at time of writing
-this spec, be able to use the export format to round-trip data. Thus,
-
-- an app may write abirtrary data into the export (given it adheres to
-  some rule)
-
-- an app may not alter or delete data written be another app (unless the
-  whole node is deleted)
-
-## Human Friendly
-
-We want to make it easy for humans to inspect and understand an export
-files' content. And, if possible, make changes on the source file.
-
-## Re-use
-
-When possible, we want to re-use existing standards and machanisms, in
-order to not reinvent the wheel.
-
-## Precise
-
-We want the spec to cleary define which OCWG files are valid, which are
-not, and especially, how to interpret the valid ones.
-
-For numeric precision, which becomes relevant in an "infinite" zoomable
-canvas, see [13](https://github.com/orgs/ocwg/discussions/13).
-
-Text rendering and container dimensions have a complex interplay. See
-[5](https://github.com/orgs/ocwg/discussions/5).
-
-## Visual Fidelity
-
-Although we don’t strive for 100% visual fideliy (see
-[11](https://github.com/orgs/ocwg/discussions/11)), there are some areas
-we need to look at:
-
-- text rendering (e.g. different browsers have different default line
-  heigths; installed fonts differ, hypenation differs, text direction,
-  Unicode, …​)
-
-- transparency & z-Index: What is in effect visible and whats hidden?
-
-- color spaces?
-
-- path stroking (rendering a path is not 100% defined)
-
-# Sample Applications
-
+<!-- TOC -->
+* [Design Decision Document](#design-decision-document)
+  * [Open Issues](#open-issues)
+  * [Sample Applications](#sample-applications)
+    * [Excalidraw](#excalidraw)
+    * [Zwibbler](#zwibbler)
+  * [Design Decisions](#design-decisions)
+    * [Nodes vs. Resources](#nodes-vs-resources)
+    * [Resources are Web-like](#resources-are-web-like)
+    * [Resource Structure](#resource-structure)
+    * [Content-Formats](#content-formats)
+    * [Schema Usage](#schema-usage)
+    * [Assets](#assets)
+    * [Relations](#relations)
+      * [Node](#node)
+      * [Group](#group)
+    * [Arrow](#arrow)
+    * [Port](#port)
+    * [Shortcut Syntax](#shortcut-syntax)
+    * [Role of JSON Schema](#role-of-json-schema)
+    * [Relations vs Visual Nodes](#relations-vs-visual-nodes)
+    * [URL and Name Space](#url-and-name-space)
+    * [Schemas as List or Map](#schemas-as-list-or-map)
+  * [References](#references)
+<!-- TOC -->
+
+## Open Issues
+-- currently none --
+
+
+## Sample Applications
 We should be able to recommend ways how to export data from some apps,
 which we are aware of.
 
-## Excalidraw
+### Excalidraw
```

Comment:
Why list Excalidraw in full and then just link to Zwibbler?

Reply:
End of time box :-) I added a TODO in the Zwibbler section.